app/src/main/java/fr/free/nrw/commons/upload/worker/UploadWorker.kt

```diff
@@ -290,7 +290,7 @@ class UploadWorker(var appContext: Context, workerParams: WorkerParameters) :
                         //Upload the file from stash
                             var countryCode: String? =null
                             with(contribution.wikidataPlace?.location){
-                                if(contribution.wikidataPlace?.isMonumentUpload!!) {
+                                if (contribution.wikidataPlace?.isMonumentUpload == true) {
```

Comment:
Would `(contribution.wikidataPlace?.isMonumentUpload)` work too?

Reply:
Hey Nicolas, contribution.wikidataPlace?.isMonumentUpload is Nullable. We would either have to use null assertion or an if else.